### PR TITLE
Fixing the OBA edit file

### PR DESCRIPTION
**Pull request by matentzn**
- removing baked in RO assumptions
- removing some noisy properties
- Changing the pav / contributor relationships to `relationship` which is what OBO wants

@anitacaron I added you in cc, maybe you see something you find odd (its to do with our orcid issue)

### CODE REVIEW

File: src/ontology/oba-edit.obo [hunk 1]
```diff
@@ -881,22 +876,22 @@ creation_date: 2014-06-04T19:27:15Z
 [Term]
 id: OBA:1000083 ! nucleus length
 synonym: "nucleus length trait" EXACT [GOC:TermGenie]
+relationship: http://purl.org/dc/elements/1.1/contributor http://orcid.org/0000-0002-2908-3327
 created_by: annethessen
```
Comment (anitacaron) [inline src/ontology/oba-edit.obo:880]: No need to replace `created_by` with `dcterms:contributor` in this ontology?

Comment (matentzn) [reply src/ontology/oba-edit.obo:880]: Yes we need to, but we are not there yet :D Thanks for the comment!

File: src/ontology/oba-edit.obo [hunk 2]
```diff
@@ -93,12 +72,12 @@ id: OBA:0000036 ! body fluid levels
...
-id: OBA:0000037 ! brood size
+id: OBA:0000037
 name: obsolete brood size
 is_a: OBA:0000001 ! biological attribute
 
 [Term]
-id: OBA:0000039 ! buoyancy
+id: OBA:0000039
 name: obsolete buoyancy
 is_a: OBA:0000001 ! biological attribute
```
Comment (dosumis) [inline src/ontology/oba-edit.obo:82]: If this is obsolete, don't we need to remove the classification?

File: src/ontology/oba-edit.obo [hunk 3]
```diff
@@ -1290,37 +1285,11 @@ id: OBA:1001088
...
-holds_over_chain: inheres_in part_of
-
-[Typedef]
-id: inheres_in ! inheres in
-is_a: inheres_in_part_of
-xref: RO:0000052 ! inheres in
-
-[Typedef]
-id: http://purl.org/pav/providedBy ! provided by
+id: http://purl.org/dc/elements/1.1/contributor ! contributor
```
Comment (dosumis) [inline src/ontology/oba-edit.obo:1289]: add `is_metadata_tag: true`

**Review by dosumis — CHANGES_REQUESTED** [review]
Couple of small notes.  Any idea if cleanup fixes Protege loading?

File: src/ontology/oba-edit.obo (hunk 3 above)
Comment (matentzn) [reply src/ontology/oba-edit.obo:1289]: That is there, just a bit.. lower down in the github diff :P

**Comment by matentzn** [comment]
Yes, that is what I did. It fixes Protege loading.

File: src/ontology/oba-edit.obo (hunk 2 above)
Comment (matentzn) [reply src/ontology/oba-edit.obo:82]: Oops some qc seems to be not running! FIxing that now.

**Comment by matentzn** [comment]
@dosumis There were actually some (minor) bugs in the TSV files, because QC never kicked in!! (robot report was set to fail on NONE) - `Bug`. Now it seems ok - your Mondo classes went in as well.

**Review by dosumis — APPROVED** [review]

File: src/ontology/oba-edit.obo (hunk 1 above)
Comment (anitacaron) [reply src/ontology/oba-edit.obo:866]: What about updating the links to HTTPS instead of HTTP?

Comment (anitacaron) [reply src/ontology/oba-edit.obo:866]: never mind 😄

**Pull request merged**